package.json

```diff
@@ -39,8 +39,8 @@
     "build-storybook": "build-storybook"
   },
   "peerDependencies": {
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0"
```

Comment:
Downgraded from `^18.2.0` to `^18.0.0` since the two are compatible with each other, with `^18.0.0` being the lowest possible compatible version before breaking changes.